integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java

```diff
@@ -397,6 +625,45 @@ private void verifyDomainNotRunning(String domainNamespace) {
     }
   }
 
+  private static void verifyMemberHealth(String adminServerPodName, List<String> managedServerNames,
```

Comment:
move this into an util class

Reply:
This method uses a specific app deployed in the test method. Its not a member for util class.